### ISSUE TITLE
Custom routes arent fully working

### ISSUE THREAD

**Issue by natenatters** [24Slides/laravel-saml2#54]
When saml2.useRoutes = false, and your new routes dont use the Tenant.uuid, but something like:
my-tenant.app.com

Then, the OneLoginBuilder doesnt respect the new URLs, it always passes the UUID. This causes issues with the IdP, because Laravel then appends the uuid like: my-tenant.app.com?uuid=123e4567-e89b-12d3-a456-426614174000

As a work around, I have replaced the usage of this class (Only referenced in ResolveTenant) and re-built it myself.

I wonder if you would be open to a discussion/ a PR around having a new way to resolve tenants based on a saml_tenants.sub_domain value.

The ideal setup would be to use it from a custom tenant model (https://github.com/24Slides/laravel-saml2/pull/49)!

**Comment by johnotaalo** [comment]
Were you able to solve this?

**Comment by natenatters** [comment]
Yeah it all works now, using the approach above. Still feels a bit hacky to implement my own code on this package, but it doesnt seem too active at the moment...
